tests/security/stig/oscap_xccdf_eval_remote.pm

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test "stig" hardening in the "scap-security-guide": detection mode with remote
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#93886, poo#104943
+
+use base 'stigtest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use bootloader_setup qw(add_grub_cmdline_settings);
+use power_action_utils "power_action";
+
+sub run {
+    my ($self) = @_;
+
+    add_grub_cmdline_settings('ignore_loglevel', update_grub => 1);
+    power_action('reboot', textmode => 1);
+    $self->wait_boot(textmode => 1);
```

Comment:
@lilyeyes 
Kindly reminder, please conside ppc64le to handle reconnect_console issue

Reply:
I tried 2 more times, test case can pass on s390x. so keep the code unmodified.